src/packet/Publish.php

```diff
@@ -135,13 +133,13 @@ protected function getVariableHeader()
         return $this->getLengthPrefixField($this->topic);
     }
 
-    /**
-     * @param \DateTime $dateTime
-     */
-    private function setReceiveTimestamp($dateTime)
-    {
-        $this->receiveTimestamp = $dateTime;
-    }
+//    /**
```

Comment:
Yep. I removed the timestamp, as it makes testing harder as the timestamp can not be predicted.
Removing it means that you can directly compare two `Publish` packet objects.

I wasn't sure you'd be happy with it, but if you are I'll get rid of it.